sensorprocess/testhelper.go

```diff
@@ -120,19 +123,55 @@ func onlineModeLidarTestHelper(
 	}
 }
 
-func onlineModeIMUTestHelper(
+func invalidAddLidarReadingInOnlineTestHelper(
```

Comment:
[q] what makes this function invalid? The addLidarReadingFunc returns a nil err every time

Reply:
I'd prefer to remove the invalid if a good lidar COULD be passed into the function and result in valid information. That reduces the confusion for:

- `invalidAddLidarReadingInOnlineTestHelper(invalidLiDAR) -> invalid result`
- `invalidAddLidarReadingInOnlineTestHelper(validLiDAR) -> valid result <-------`

to

- `addLidarReadingInOnlineTestHelper(invalidLiDAR) -> invalid result`
- `addLidarReadingInOnlineTestHelper(validLiDAR) -> valid result`

Reply:
It has to be "invalid" because it actually ensures that things fail. There's another, similar function, where we check that it's valid. Won't change anything now since refactoring this test is not part of the scope for this ticket, but will keep it in mind when I work on integration tests.